[0m23:57:20.753652 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000018384EA0DD0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000018384EA1090>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000018384EA1C90>]}


============================== 23:57:20.762492 | 7caddbaa-ecf4-4669-b764-08f3b4eac3e5 ==============================
[0m23:57:20.762492 [info ] [MainThread]: Running with dbt=1.7.1
[0m23:57:20.762492 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\ausje\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'warn_error': 'None', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'log_format': 'default', 'introspect': 'True', 'invocation_command': 'dbt deps', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'static_parser': 'True', 'send_anonymous_usage_stats': 'True'}
[0m23:57:20.762492 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\ausje\OneDrive\PC storage\Github\dbt-dimensional-modelling\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m23:57:20.769450 [debug] [MainThread]: Command `dbt deps` failed at 23:57:20.769450 after 0.08 seconds
[0m23:57:20.770511 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000018384EA06D0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x00000183FDD5BC50>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000018384EEC150>]}
[0m23:57:20.770511 [debug] [MainThread]: Flushing usage events
